packages/hol2dk/hol2dk.1.0.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "HOL-Light to Dedukti/Lambdapi and Coq translator"
+description: "HOL-Light to Dedukti/Lambdapi and Coq translator"
+maintainer: ["Frédéric Blanqui"]
+authors: ["Frédéric Blanqui" "Anthony Bordg"]
+license: "CeCILL-2.1"
+homepage: "https://github.com/Deducteam/hol2dk"
```

Comment:
it seems that the `doc:` field is removed. please confirm whether this is intentional
https://github.com/ocaml/opam-repository/blob/0758a2c8add170e79d41976d4a9d2991dc0f7686/packages/hol2dk/hol2dk.0.0.1/opam#L8

Reply:
Yes, indeed, because the opam file is generated by dune but I get an error with dune-release. I don't know why.

Reply:
```suggestion
homepage: "https://github.com/Deducteam/hol2dk"
doc: "https://github.com/Deducteam/hol2dk/blob/master/README.md"
```

Reply:
@fblanqui  I see. A `(document ..)` field in your `dune-project` file might help.

ref: https://dune.readthedocs.io/en/stable/dune-files.html#stanza-documentation